.travis.yml

```diff
@@ -41,6 +41,7 @@ before_install:
     fi
 
 install:
+  - npm install --global npm@5
```

Comment:
Can we only perform this step on OS X builds, so we don't do it twice on GNU/Linux?